### ISSUE TITLE
need password option for TextBox

### ISSUE THREAD

**Issue by maurerdietmar** [linebender/druid#2055]
Or is there already a way to hide passwords?

**Comment by ngugcx** [comment]
There is PR(#1839 ), but it's closed without merged.

**Comment by maurerdietmar** [comment]
> There is PR(#1839 ), but it's closed without merged.

Yes, I saw that. But I do not understand why it is closed. This is an important Feature, and looks quite simple to implement...

**Comment by Maan2003** [comment]
it is not simple to handle all the cases :sweat_smile:

**Comment by ngugcx** [comment]
> > There is PR(#1839 ), but it's closed without merged.
> 
> Yes, I saw that. But I do not understand why it is closed. This is an important Feature, and looks quite simple to implement...

I also do not understand. Many important features are not supported, but no one cares or just lack of resources. Application developers are considered to build all the necessary widgets themselves.
